src/embezzler.ts

```diff
@@ -261,14 +277,15 @@ export const embezzlerSources = [
       ).step(baseMacro);
       adventureMacro($location`Noob Cave`, macro);
     },
-    [new Requirement([], { forceEquip: $items`Powerful Glove` })]
+    [new Requirement([], { forceEquip: $items`Powerful Glove` })],
+    { swaps: true }
   ),
   new EmbezzlerFight(
     "Be Gregarious",
     () =>
       retrieveItem(1, $item`human musk`) &&
       get("beGregariousMonster") === $monster`Knob Goblin Embezzler` &&
-      get("beGregariousFightsLeft") > 0,
+      get("beGregariousFightsLeft") > (have($item`miniature crystal ball`) ? 1 : 0),
```

Comment:
I know what this is intended to do, but remove it; we need to rehaul orb code to match the new mafia system

Reply:
Yes, I did this as a short term fix because I'm on old mafia